RecoParticleFlow/PFProducer/plugins/PFEGammaProducer.cc

```diff
@@ -197,6 +197,7 @@ void PFEGammaProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup
       switch (elements[0].type()) {
         case reco::PFBlockElement::SC:
           edm::LogError("PFEGammaProducer") << "PFBLOCKALGO BUG!!!! Found a SuperCluster in a block by itself!";
+          [[fallthrough]];
```

Comment:
I would rather ```break``` here: @hatakeyamak @bendavid do you agree ?

Reply:
@hatakeyamak @bendavid do you agree?

Reply:
ping

Reply:
break will yield the correct behavior still since the SuperCluster block element is alone in the block to begin with. Probably better to do that.

Reply:
Oops. Somehow, I didn't notice this ping from this PR until now.
The "break" as suggested by @perrotta and @lgray sounds reasonable to me.